src/components/pages/farmer-member/list/modals/multi-marker-map/geojson-multi-marker-map.tsx

```diff
@@ -0,0 +1,126 @@
+import MarkerClusterGroup from "@changey/react-leaflet-markercluster";
+import L, { divIcon, latLngBounds } from "leaflet";
+import React, { useEffect } from "react";
+import { GeoJSON, LayerGroup, LayersControl, MapContainer, TileLayer, useMap } from "react-leaflet";
+
+export default function FarmerMap({ geojsonData }) {
+  const mapStyle = {
+    width: "100%",
+    height: "100%",
+  };
+
+  const calculateBounds = (geojsonData) => {
+    const bounds = latLngBounds();
+
+    geojsonData.forEach((feature) => {
+      const { type, coordinates } = feature.geometry;
+
+      switch (type) {
+        case "Point":
+          bounds.extend([coordinates[1], coordinates[0]]);
+          break;
+        case "MultiPoint":
+          coordinates.forEach((coord) => {
+            bounds.extend([coord[1], coord[0]]);
+          });
+          break;
+        // Add cases for other geometry types if needed
+        default:
+          console.warn(`Unsupported geometry type: ${type}`);
+      }
+    });
+
+    return bounds;
+  };
+
+  const createColoredIcon = (color) => {
```

Comment:
extract out this icon inside `src/icons` and then use it here

Reply:
As discussed in the meeting, this is being only used once and it is different from normal icons. So moving forward without moving.